test/Datadog.Trace.ClrProfiler.IntegrationTests/HttpClientTests.cs

```diff
@@ -22,6 +22,10 @@ public HttpClientTests(ITestOutputHelper output)
         [Trait("RunOnWindows", "True")]
         public void HttpClient()
         {
+            int expectedSpanCount = EnvironmentHelper.IsCoreClr() ? 2 : 1;
+            const string expectedOperationName = "http.request";
+            const string expectedServiceName = "Samples.HttpMessageHandler-http-client";
```

Comment:
@zacharycmontoya Is there any way to distinguish a `SocketHttpHandler` from another `HttpMessageHandler` request?

Reply:
In the final span that's created, the `http-client-handler-type` tag will be `System.Net.Http.HttpClientHandler` instead of `System.Net.Http.HttpClientHandler`. Otherwise, they're pretty indistinguishable and we only generate a span for the first handler in the chain.

Reply:
Er, did you mean `SocketsHttpHandler`?

Reply:
Whoops! Yes thanks for pointing that out, I've updated my previous comment now